src/Umbraco.Infrastructure/Persistence/NPocoSqlExtensions.cs

```diff
@@ -845,6 +845,21 @@ public static Sql<ISqlContext> Select<TDto>(this Sql<ISqlContext> sql, Func<SqlR
             return sql;
         }
 
+        /// <summary>
+        /// Creates a SELECT CASE WHEN EXISTS query, which returns 1 if the sub query returns any results, and 0 if not
+        /// </summary>
+        /// <param name="sql">The origin SQL</param>
+        /// <param name="nestedSelect">The nested select to run the query against</param>
+        /// <returns></returns>
+        public static Sql<ISqlContext> SelectAnyExists(this Sql<ISqlContext> sql, Sql<ISqlContext> nestedSelect)
```

Comment:
Maybe rename `SelectAnyExists` to <code>SelectAny<b>If</b>Exists</code>?

Reply:
Good idea :D, naming is hard :)